packages/components/lib/utils/platform_utils.dart

```diff
@@ -0,0 +1,32 @@
+import 'dart:io' show Platform;
+import 'package:flutter/foundation.dart' show kIsWeb;
+
+/// Source : https://stackoverflow.com/questions/58459483/unsupported-operation-platform-operatingsystem/75393239#75393239
+/// A class that returns the platform on which the application is running
```

Comment:
```suggestion
/// A class that returns the platform on which the application is running
///
/// Source : https://stackoverflow.com/questions/58459483/unsupported-operation-platform-operatingsystem/75393239#75393239
```

Just change for style